### PR TITLE
several small fixes/optimizations for quick hosting pre kickoff

### PR DESCRIPTION
**Pull request by Jared-Krajewski**
## This PR:

No issue for this PR. Its essentially some last minute changes to have this hosted for tomorrow. Feedback on easy/essential things is still welcome but further significant changes can come through a final issue creation session.

---

## Future Steps/PRs Needed to Finish This Work (optional):

This is by no means to get this perfect. Just close enough where we can host and display this tomorrow.

### CODE REVIEW

File: src/components/home/about/imageList.js [hunk 1]
```diff
@@ -12,8 +12,8 @@ export const imageList = [
     alt: 'head shot of Ka Hung, image 3 out of 5'
   },
   {
-    image: '/assets/volunteers/code-pdx-volunteer-andy.webp',
-    alt: 'head shot of Andy, image 4 out of 5 '
```
Comment (andycwilliams) [inline src/components/home/about/imageList.js:16]: Wow. Okay. Rude.

File: src/components/global/Footer.jsx [hunk 2]
```diff
@@ -13,20 +13,20 @@ import { FaDiscord, FaGithub, FaLinkedin, FaMeetup } from 'react-icons/fa6';
...
-  },
-  {
-    href: 'https://www.codepdx.org/',
+    href: 'https://github.com/codeforpdx/codeofconduct',
     title: 'Code of Conduct'
   },
   {
-    href: 'https://www.codepdx.org/',
+    href: 'https://docs.google.com/forms/d/e/1FAIpQLSfu4u9FB00RXd2TncbMLws_Ckot-vYqPK0oUHr81uLDjThE7Q/viewform',
     title: 'Contact Us'
```
Comment (ogorman89) [inline src/components/global/Footer.jsx]: maybe we title this 'sign up' since we have 'contact us' in other places that trigger the contact model.

**Review by ogorman89 — APPROVED** [review]
Looks good. 🚀

File: src/components/global/Footer.jsx (hunk 2 above)
Comment (Jared-Krajewski) [reply src/components/global/Footer.jsx]: I'll change that. good idea. I wasn't really sure what to do so I panicked and linked to the same page.

**Pull request merged**